src/smart-components/OpenChannelApp/streaming.scss

```diff
@@ -70,21 +70,23 @@
   .chat-panel {
     background-color: var(--bg-0);
     width: 320px;
-    .sendbird-openchannel-thumbnail-message
-    .sendbird-openchannel-thumbnail-message__right
-    .sendbird-openchannel-thumbnail-message__right__body
     .sendbird-openchannel-thumbnail-message__right__body__wrap {
       .sendbird-openchannel-thumbnail-message__right__body__wrap__overlay,
       .sendbird-openchannel-thumbnail-message__right__body__wrap__video,
       .sendbird-openchannel-thumbnail-message__right__body__wrap__video__video {
-        max-width: 95%;
+        max-width: 95% !important;
```

Comment:
using 100% is better I think?
<img width="304" alt="image" src="https://user-images.githubusercontent.com/46333979/178179090-59a9fa1e-43c4-4abe-a900-abfff4ee8d68.png">

Reply:
can you chnage and merge?